airbyte-integrations/bases/base-singer/base_singer/singer_helpers.py

```diff
@@ -60,6 +60,15 @@ def _transform_types(stream_properties: DefaultDict):
             field_object = stream_properties[field_name]
             field_object["type"] = SingerHelper._parse_type(field_object["type"])
 
+    @staticmethod
+    def singer_catalog_to_airbyte_catalog(singer_catalog: Dict[str, str]) -> AirbyteCatalog:
```

Comment:
How of curiosity, why did you decide to split that method?

Reply:
At some point during implementation I wanted to read a singer catalog and convert it to an Airbyte catalog so I extracted it. I realized I could accomplish that task in a simpler way so I didn't end up doing it. It's not strictly necessary to do the refactor now but I think this is a useful standalone method so I kept it.